### PR TITLE
Add emoticons to SVG database

### PR DESCRIPTION
**Pull request by VoxelDoesCode**
NOTE: This might not be perfect, especially the music notes, but it doesn't need to be pixel perfect, right?

### CODE REVIEW

**Comment by ChillerDragon** [comment]
Original:
image

Svg:
image

That could be done a bit better :p

**Comment by def-** [comment]
Font not embedded probably. Could also convert it so no font is required.

File: assets/emoticons.svg [hunk 1]
```diff
@@ -0,0 +1,594 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   width="135.46666mm"
```
Comment (ChillerDragon) [inline assets/emoticons.svg:5]: Would be nice if the width and height were in pixel not in millimeter and match the resolution of the original image.
So 512 x 512 in this case.

**Comment by Jupeyy** [comment]
> Nice, with assets/ was correct BTW. It has to be the same file as emoticons.png in ddnets data directory

since this is out of context now, i think u asked stuff on dc.

It has to be the same file as emoticons.png in ddnet's data directory, so it has to live next to game.png which is in the root directory

**Pull request merged**